CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Unreleased as of Sprint 73 ending 2017-11-13
+
+### Added
+- Allow for substitution of hosts  [(#102)](https://github.com/ManageIQ/manageiq-automation_engine/pull/102)
```

Comment:
Need to make this one more descriptive: `Allow for substitution of hosts in Ansible Playbook methods`